### ISSUE TITLE
Pictureboxes are jagged when zoomed too far in/out

### ISSUE THREAD

**Issue by ricleichtung** [udbhav/adhoc#55]

**Comment by udbhav** [comment]
This is a result of the way that I must format it to have all 3 boxes the same height.  If you're ok with the boxes being different heights depending on the amount of content in them, then the boxes will not be jagged when zooming.

**Comment by ricleichtung** [comment]
yeah, maybe choosing a smaller font for these posts, or the ability to alter what appears in the boxes would be good-- def can't be jagged
